lib/internal/process/pre_execution.js

```diff
@@ -554,6 +554,22 @@ function initializePermission() {
         'It could invalidate the permission model.', 'SecurityWarning');
       }
     }
+    const fsReadValue = getOptionValue('--allow-fs-read');
+    if (fsReadValue.length === 1 && (fsReadValue[0] === '*' || fsReadValue[0] === '/')) {
+      process.emitWarning(
+        'Granting all to --allow-fs-read leaks all sensitive info on the host machine.',
+        'SecurityWarning'
+      );
+    }
+    const fsWriteValue = getOptionValue('--allow-fs-write');
+    if (fsWriteValue.length === 1 && (fsWriteValue[0] === '*' || fsWriteValue[0] === '/')) {
```

Comment:
This does not apply to Windows I think.

Reply:
Yeah, Windows is a nightmare. I guess there are many ways to refer to the entire file system namespace, e.g., some UNC path prefix with a wildcard? Perhaps it's best to only warn about `*` on Windows.